### DISCUSSION TITLE
About the PixArt-α-SAM-256 Checkpoint

### DISCUSSION THREAD

**Discussion by hyang0511** [PixArt-alpha/PixArt-alpha#36] [General]
Is there any plan to release the PixArt-α-SAM-256 checkpoint?
Does this have an estimated date?

**Comment by lawrence-cj** [comment]
very near future.

**Reply by hyang0511** [reply]
Are there any updates on this?

**Reply by lawrence-cj** [reply]
Released here: https://github.com/PixArt-alpha/PixArt-alpha#-download-models
